app/src/main/res/layout/settings_tile.xml

```diff
@@ -35,7 +35,10 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:gravity="center"
-        android:padding="12dp"
```

Comment:
Each commit should follow our commit format, would you mind amending the second?

Early commits will typically be prefaced with `Issue #<issue>:`, then `[Closes|Fixes] $<issue>` when you think the PR is complete.  It's also fine if the last few commits use `Closes`, due to review comments and other late changes.

Reply:
It would also be fine to squash these into one commit.

Reply:
Hmm I'm not sure how to change a commit message that's already been pushed. Is this okay, or did you have something else in mind?

Reply:
Are you still in the office?  If not we can go over it in person on Monday.  It's a simple process but it comes with some warnings.